Create Blog Post “tji-travels-under-the-pink-dome” (#612)
texas-justice-initiative/website-nextjs: content/blog/posts/tji-travels-under-the-pink-dome.md

---
title: TJI Travels Under the Pink Dome
subtitle: ""
date: 2021-05-10
authors:
  - Eva Ruth Moravec
---

> Every other year, the pink granite building that sits in the heart of Austin buzzes with activity for six months as lawmakers gather to debate, consider, toss out and approve legislation that governs life in Texas. 

Reviewer: Picture of the building? 😝 (Just a thought)

Author: good point. I'm not sure how to add one though – should I wait until it's in the CMS? I was hoping to use one of the [Getty embeddable images ](https://www.gettyimages.com/search/2/image?phrase=texas+capitol)

Reviewer: Oh very nice. Probably @nholden might know the answer :)

> This is one of said years, and said lawmakers are currently tumbleweeding towards the finish line at the end of May, when the clock runs out for bills to make it to Gov. Greg Abbott's desk for one last vote. For the first time, I testified *for* two bills – both proposals that would enhance transparency and accountability in criminal justice. While neither is likely to become law this session, I'm grateful for the opportunity and learned a new thing or two about this sausage-making process of ours that I thought I'd share. 

Reviewer: I'm curious to know how you got to participate in the session from the beginning.

Author: another great point!

> This is one of said years, and said lawmakers are currently tumbleweeding towards the finish line at the end of May, when the clock runs out for bills to make it to Gov. Greg Abbott's desk for one last vote. For the first time, I testified *for* two bills – both proposals that would enhance transparency and accountability in criminal justice. While neither is likely to become law this session, I'm grateful for the opportunity and learned a new thing or two about this sausage-making process of ours that I thought I'd share. 

Author: ```suggestion
This is one of said years, and said lawmakers are currently tumbleweeding towards the finish line at the end of May, when the clock runs out for bills to make it to Gov. Greg Abbott's desk for one last vote. For the first time, I testified *for* two bills – one, at the urging of the Freedom of Information Foundation of Texas and the other at the behest of the bill's author, a state representative – both proposals that would enhance transparency and accountability in criminal justice. While it's unclear if either will become law this session, I'm grateful for the opportunity and learned a new thing or two about this sausage-making process of ours that I thought I'd share. 
```

It all starts  weeks before the start of the Texas legislative session, when Texas representatives and senators [start to file bills](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/). Then in January, lawmakers, lobbyists and advocates and everyday Texans raise their voices. Ambitiously, legislators file thousands of bills – [this year, some 8,829 ](https://legiscan.com/TX)– and the race is on. In order for a bill to become state law, it must be heard in and voted out of committees in both chambers, and then in both chambers' floor sessions, and finally considered by the governor. 

> I testified for two House bills: [HB 1810](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/), which addresses the format in which agencies release data, got voted out of the massive State Affairs committee and even the full House; and [HB 2901](https://capitol.texas.gov/BillLookup/History.aspx?LegSess=87R&Bill=HB2901), which would address law enforcement agencies that misreport deaths of people in their custody, stalled after a hearing in Homeland Security and Public Safety.  

Reviewer: Is it possible to summarize your testimony? It would be interesting to know what kind of questions you've gotten and especially related to TJI's activities.

> I testified for two House bills: [HB 1810](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/), which addresses the format in which agencies release data, got voted out of the massive State Affairs committee and even the full House; and [HB 2901](https://capitol.texas.gov/BillLookup/History.aspx?LegSess=87R&Bill=HB2901), which would address law enforcement agencies that misreport deaths of people in their custody, stalled after a hearing in Homeland Security and Public Safety.  

Author: ```suggestion
I testified for two House bills: [HB 1810](https://www.texastribune.org/2020/11/09/texas-bill-filing-legislative-session/), which addresses the format in which agencies release data, got voted out of the massive State Affairs committee and even the full House; and [HB 2901](https://capitol.texas.gov/BillLookup/History.aspx?LegSess=87R&Bill=HB2901), which would address law enforcement agencies that misreport deaths of people in their custody, stalled after a hearing in Homeland Security and Public Safety.  For HB 1810, I explained that at least one agency that I regularly ask for data constantly produces it in non-CSV format, which enhances the potential for human error. For HB 2901, I explained how there is no current official process to follow if a custodial death report is found to have errors or be missing altogether. 
```

> While I'd covered plenty of legislative hearings as a reporter before, I'd never spoken as a witness in one. Here's what I learned: 

Reviewer: Any COVID-related changes?

1. After a year of donning stretchy pants, wearing business attire can baffle the mind. You can't go wrong with all-black.

> 2. Two or three minutes of uninterrupted testimony can really fly by. Focus testimony on your own personal and unique experiences. 

Author: ```suggestion
2. The rules on mask-wearing in the Capitol have changed over time. Even if you're vaccinated, stash one in your pocket or bag just in case. 

3. Two or three minutes of uninterrupted testimony can really fly by. Focus testimony on your own personal and unique experiences. 
```

> 3. Other witnesses' comments may call for planned testimony to change. Be flexible and pivot as needed. 

Author: ```suggestion
4. Other witnesses' comments may call for planned testimony to change. Be flexible and pivot as needed. 
```

> 4. If you're skilled enough to peel your eyes away from  prepared remarks, don't be surprised/hurt if all of the committee members seem to be distracted or not paying attention to you. 

Author: ```suggestion
5. If you're skilled enough to peel your eyes away from  prepared remarks, don't be surprised/hurt if all of the committee members seem to be distracted or not paying attention to you. 
```

> 5. Do not expect questions (but prepare for them just in case). 

Author: ```suggestion
6. Do not expect questions (but prepare for them just in case). 
```

> 6. Do expect to think of things you wish you'd said, and chalk it up to inexperience. Next time will be better – practice makes perfect, and bills often take years to become law. 

Author: ```suggestion
7. Do expect to think of things you wish you'd said, and chalk it up to inexperience. Next time will be better – practice makes perfect, and bills often take years to become law. 
```
